c7n/resources/account.py

```diff
@@ -859,7 +859,14 @@ def get_check_result(cls, client, check_id):
 
         # Check status and if necessary refresh checks
         if checks['status'] == 'not_available':
-            client.refresh_trusted_advisor_check(checkId=check_id)
+            try:
+                client.refresh_trusted_advisor_check(checkId=check_id)
+            except ClientError as e:
+                if e.response['Error']['Code'] == 'InvalidParameterValueException':
+                    cls.log.warning("InvalidParameterValueException: " + 
+                        "The provided parameter is invalid.")
```

Comment:
We probably want to pull in `e.response["Error"]["Message"]` here rather than hardcoding a generic message.

Reply:
let me address this feedback and get back on this :)

Reply:
I enriched a bit the error message now :)